js/showdown-extensions.js

```diff
@@ -1,177 +1,181 @@
 // function allowing to convert some text to its slug
 function wbSlugify(obj) {
-    var text = '';
-    if (typeof obj === 'string') {
-        text = obj;
-    } else if (obj instanceof HTMLElement) {
-        text = obj.textContent;
-    } else {
-        console.error("wbSlugify: Unsupported input");
-    }
-    return text.
-        trim().
-        toLowerCase().
-        replace(/[\s\.]/g, '-').
-        replace(/[-]+/g, '-').
-        replace(/^-*/, '').
-        replace(/-*$/, '').
-        replace('+', 'p').
-        replace(/[^\w-]+/g, '');
+  var text = '';
+  if (typeof obj === 'string') {
+    text = obj;
+  } else if (obj instanceof HTMLElement) {
+    text = obj.textContent;
+  } else {
+    console.error('wbSlugify: Unsupported input');
+  }
+  return text.
+    trim().
+    toLowerCase().
+    replace(/[\s\.]/g, '-').
+    replace(/[-]+/g, '-').
+    replace(/^-*/, '').
+    replace(/-*$/, '').
+    replace('+', 'p').
+    replace(/[^\w-]+/g, '');
 }
 
 // This extension is a template-like mechanism, allowing
 // to replace variables by a static content.
-// For example, the markdown string "{{ date.year }}" is replaced by "2016"
-showdown.extension("wbVariables", function() {
+// For example, the markdown string '{{ date.year }}' is replaced by '2016'
+showdown.extension('wbVariables', function() {
     // static variables to maintain
     // TODO: could be computed
     var vars = {
       webots : {
         version : {
-          major : "R2018b",
+          major : 'R2018b',
           // full is equal to major for the first major version
           // and contains the revision number for subsequent versions
-          full : "R2018b",
-          package : "R2018b"
+          full : 'R2018b',
+          package : 'R2018b'
         }
       },
       date : {
         year : 2018
       }
-    };
-    // compute debian package version format by removing initial 'R'
-    vars.webots.version.debian_package = vars.webots.version.package.substring(1);
+    },
+    date : {
+      year : 2018
+    }
```

Comment:
Sorry to see this only now (because it probably comes from your previous merge) but it seems there is a problem here, the date is defined twice.

Reply:
ach, well spotted: it's fixed here: https://github.com/omichel/webots-doc/pull/609